frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -37,11 +37,11 @@ action.GitMachete.PullCurrentBranchFastForwardOnlyBackgroundable.operation-name=
 
 # Base actions
 
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward branch ''{0}'' to match ''{1}''
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward of branch ''{0}'' to match ''{1}'' failed
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-success=Branch <b>{0}</b> has been fast-forwarded to match <b>{1}</b>
-action.GitMachete.BaseFastForwardParentToMatchBranchAction.task-title=Fast Forwarding
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.description-action-name=Fast forward merge
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.description=Fast forward merge of branch ''{1}'' to ''{0}''
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-fail=Fast forward merge of branch ''{1}'' to ''{0}'' failed
+action.GitMachete.BaseFastForwardParentToMatchBranchAction.notification.title.ff-success=Branch <b>{1}</b> has been fast-forward merged to <b>{0}</b>
```

Comment:
UX micro-optimization: I'd use the adjective notation (`fast-forward`, `Fast-forward` &mdash; with a hyphen) everywhere for clarity, it makes it clear that the words "fast forward" describe the word "merge" :thinking: